spec/models/claims/mentor_membership_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe Claims::MentorMembership, type: :model do
+  context "with associations" do
+    it { is_expected.to belong_to(:mentor) }
+    it { is_expected.to belong_to(:school) }
```

Comment:
Would it be valuable to test/show that the returned associations are all within the `Claims::` namespace?

I think that's the reason why we re-declare the association in each sub-class, so that they pick up the namespace. Right? A test here would cover the re-defining of those associations.

e.g. so `Claims::MentorMembership#school` will always return a `Claims::School`, and `Claims::MentorMembership#mentor` will always return a `Claims::Mentor`.